curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md

```diff
@@ -0,0 +1,87 @@
+---
+id: 6338628e24620bf3848eb082
+title: Step 42
+challengeType: 0
+dashedName: step-42
+---
+
+# --description--
+
+step 41 instructions
```

Comment:
This is incomplete

Reply:
Step 42 has not been added by me. I only added Step 41 in the editor.

Reply:
You have mistakenly added two steps here

Reply:
It is incrementing the steps by 2, even though I am adding only one step. Can you please tell me what I can do to fix this? I worked on it last night, and I cannot seem to fix it.

Reply:
I am not an expert with the challenge editor. All I know is that you added two additional files, as shown in the list of changes.